src/main/java/com/qingstor/sdk/constants/QSConstant.java

```diff
@@ -79,16 +79,18 @@
     public static String LOGGER_LEVEL = "error";
 
     public static String LOGGER_ERROR = "error";
-    public static String LOGGER_WARNNING = "warn";
+    public static String LOGGER_WARNING = "warn";
     public static String LOGGER_INFO = "info";
     public static String LOGGER_DEBUG = "debug";
     public static String LOGGER_FATAL = "fatal";
 
-
     public static int HTTPCLIENT_CONNECTION_TIME_OUT = 60; // Seconds
     public static int HTTPCLIENT_READ_TIME_OUT = 100; // Seconds
     public static int HTTPCLIENT_WRITE_TIME_OUT = 100; // Seconds
 
-    public static final String VIRTUAL_HOST_STYLE  = "virtual_host_style"; //default url style, like this: https://bucket-name.zone-id.qingstor.com/object-name
-    public static final String PATH_STYLE = "path_style"; //https://zone-id.qingstor.com/bucket-name/object-name
+    public static final String VIRTUAL_HOST_STYLE =
+            "virtual_host_style"; // default url style, like this:
```

Comment:
code style format?

Reply:
Yes, we use spotless format all .java file in source dir. Format style use google-java-format's AOSP style(use 4 indents).